llvm/lib/Object/SymbolSize.cpp

```diff
@@ -59,6 +59,13 @@ llvm::object::computeSymbolSizes(const ObjectFile &O) {
     return Ret;
   }
 
+  if (const auto *E = dyn_cast<XCOFFObjectFile>(&O)) {
+    auto Syms = E->symbols();
```

Comment:
This local variable is unnecessary. Just put it directly into the for loop sequence.